pages/common/pbzip2.md

```diff
@@ -0,0 +1,20 @@
+# pbzip2
+
+> A parallel implementation of the bzip2 file compressor.
+> More information: <http://compression.great-site.net/pbzip2/>.
+
+- Compress a file:
+
+`pbzip2 {{path/to/file_to_compress}}`
+
+- Compress a file using 4 processors:
+
+`pbzip2 -p4 {{path/to/file_to_compress}}`
+
+- Decompress a file:
+
+`pbzip2 -d {{path/to/compressed_file.bz2}}`
```

Comment:
```suggestion
- [d]ecompress a file:

`pbzip2 --decompress {{path/to/compressed_file.bz2}}`
```
From the style-guide:
> Use GNU-style long options (like --help rather than -h) when they are cross-platform compatible (intended to work the same across multiple platforms).

> Short option mnemonics are optional hints which can be added to help users understand the meaning of these short options. The assigned mnemonics should match with the ones in the command's official documentation (e.g. from man or Get-Help)